include/z64animation.h

```diff
@@ -9,23 +9,24 @@
 struct PlayState;
 struct Actor;
 struct SkelAnime;
+struct PlayerAnimationFrame;
 
 #define LINK_ANIMETION_OFFSET(addr, offset) \
-    (SEGMENT_ROM_START(link_animetion) + ((u32)addr & 0xFFFFFF) + ((u32)offset))
+    (SEGMENT_ROM_START(link_animetion) + ((uintptr_t)addr & 0xFFFFFF) + ((uintptr_t)offset))
```

Comment:
```suggestion
    (SEGMENT_ROM_START(link_animetion) + ((uintptr_t)addr & 0xFFFFFF) + ((u32)offset))
```
Small, but I don't think I would consider the offset as an actual pointer.